### PR TITLE
Auto Save SS Pause, Save Screen Overlay.

### PR DESCRIPTION
**Pull request by SirBloodBuske**
Art/Sprites made by @NataKilar 
image

### CODE REVIEW

File: code/controllers/autosave.dm [hunk 1]
```diff
@@ -16,12 +19,25 @@ var/datum/controller/save_controller/save_controller
...
+		anounce1 = 1
+		to_world("<span class='notice'>Routine Neural Lace Diagnostics in 5 Minutes.</span>")
 
+	if (!anounce2 && time_till_autosave() <= 1 MINUTE)
+		anounce2 = 1
+		to_world("<span class='notice'>Routine Neural Lace Diagnostics in 1 Minute.</span>")
+
+	if (!anounce3 && time_till_autosave() <= 10 SECOND)
+		anounce3 = 1
+		to_world("<font size=4 color='green'>Routine Neural Lace Diagnostics in 10 Seconds.</font>")
```
Comment (ingles98) [inline code/controllers/autosave.dm]: The <font> tag is pretty much deprecated and non existent on HTML5. Shouldn't be an issue as long as BYOND doesn't get us HTML5.

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -35,8 +35,8 @@ var/list/gamemode_cache = list()
 	var/vote_autogamemode_timeleft = 100 //Length of time before round start when autogamemode vote is called (in seconds, default 100).
 	var/vote_no_default = 0				// vote does not default to nochange/norestart (tbi)
 	var/vote_no_dead = 0        		// dead people can't vote (tbi)
-	var/autosave_initial =  3 HOUR		//Length of time before the first autoSave
```
Comment (ingles98) [inline code/controllers/configuration.dm]: File not changed, diffed white spaces.
Remove this change

File: code/game/gamemodes/save_mode.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+	//create the cinematic screen obj
+	icon = 'icons/effects/gateway_intro.dmi'
+	icon_state = "savescreen"
+	plane = HUD_PLANE
+	layer = HUD_ABOVE_ITEM_LAYER
+	mouse_opacity = 2
+	screen_loc = "WEST,SOUTH"
+
+/datum/universal_state/save/proc/start_cinematic_intro()
+	for(var/mob/M in GLOB.player_list) //I guess so that people in the lobby only hear the explosion
```
Comment (Andrew-Fall) [inline code/game/gamemodes/save_mode.dm]: This doesn't appear to do anything

Comment (SirBloodBuske) [reply code/game/gamemodes/save_mode.dm]: Fixed sorry about that.

Comment (Andrew-Fall) [reply code/game/gamemodes/save_mode.dm]: It doesn't appear to be fixed, unless you have yet to update your git to the latest

**Pull request closed without merging**